cmd/flags_test.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFlagEqualityAgainstSDK(t *testing.T) {
```

Comment:
Wouldn't hurt to have a comment here explaining the intent of the flags being equal without the `cmd` package directly depending on `sdk/client/flags`.